cni/network/network.go

```diff
@@ -205,6 +229,11 @@ func (plugin *netPlugin) Add(args *cniSkel.CmdArgs) error {
 
 	log.Printf("[cni-net] Read network configuration %+v.", nwCfg)
 
+	// Temporary if block to determing whether we disable SNAT on host
+	if err = determineSnatOnHost(nwCfg); err != nil {
```

Comment:
isn't nwCfg.EnableSnatOnHost gets overridden after cni add function call?

Reply:
not sure what you mean here...